### PR TITLE
CON-2434 resume migration to V2

### PR DESCRIPTION
**Pull request by shomavg**

### CODE REVIEW

**Review by hkf-tech — CHANGES_REQUESTED** [review]

**Review by hkf-tech — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Apprentice/ChangeStatus.cshtml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+            <div class="govuk-form-group  @(ViewData.ModelState.IsValid ? " " : " govuk-form-group--error")">
+                <fieldset class="govuk-fieldset">
+                    @if (!ViewData.ModelState.IsValid && ViewData.ModelState.ContainsKey(nameof(ChangeStatusRequestViewModel.SelectedStatusChange)))
+                    {
+                    <span id="error-message-@nameof(ChangeStatusRequestViewModel.SelectedStatusChange)" class="govuk-error-message">
+                        <span class="govuk-visually-hidden">Error:</span> @ViewData.ModelState[nameof(ChangeStatusRequestViewModel.SelectedStatusChange)].Errors.First().ErrorMessage
+                    </span>
+                    }
+                <div class="govuk-radios">
```
[the remark this answers is not in the document]
Comment (shomavg) [reply src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Apprentice/ChangeStatus.cshtml:49]: It is working as mentioned in the AC's.

[the remark this answers is not in the document]
Comment (hkf-tech) [reply src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Apprentice/ChangeStatus.cshtml:49]: Pause radio button is not visible in the AC, but please let me know if I am missing something.

File: src/SFA.DAS.EmployerCommitmentsV2.Web.UnitTests/Controllers/ApprenticeControllerTests/WhenRequestingResumeApprenticeshipConfirmationPage.cs [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+        [Test]
+        public async Task AndCurrentStatusIsPaused_ThenViewIsReturned()
+        {
+            _mockModelMapper.Setup(m => m.Map<ResumeRequestViewModel>(It.IsAny<ResumeRequest>()))
+                .ReturnsAsync(new ResumeRequestViewModel());
+
+            var result = await _controller.ResumeApprenticeship(new ResumeRequest());
+
+            Assert.IsInstanceOf<ViewResult>(result);
```
Comment (hkf-tech) [inline src/SFA.DAS.EmployerCommitmentsV2.Web.UnitTests/Controllers/ApprenticeControllerTests/WhenRequestingResumeApprenticeshipConfirmationPage.cs:34]: var viewResult = (ViewResult)result;
                var viewModel = viewResult.Model;

                Assert.IsInstanceOf<ResumeRequestViewModel>(viewModel);
                var resumeRequestViewModel= (ResumeRequestViewModel) viewModel;

                Assert.AreEqual(_viewModel, resumeRequestViewModel);

**Review by hkf-tech — APPROVED** [review]
Good job.

**Pull request merged**